### PR TITLE
`casegen_upcars` Add more detail documentation

### PR DESCRIPTION
**Pull request by alifbe**
Improve documentation page for `casegen_upcars`

### CODE REVIEW

**Review by rnyb — APPROVED** [review]
Nice

File: docs/scripts/casegen_upcars.rst [hunk 1]
```diff
@@ -2,6 +2,268 @@
 CASEGEN_UPCARS
 ==============
 
+casegen_upcars is a script to generate simple conceptual models at different
+scales, containing different types of heterogeneity e.g. matrix, fracture, vug,
+fault, super-K layers etc. The script is developed as part of UpCaRs (Upscaling
+in complex carbonate reservoirs) project.
+
+Output from casegen_upcars is in the form of Eclipse simulation models. These
+models can be run to investigate impact of changing geological properties on
```
Comment (mferrera) [inline docs/scripts/casegen_upcars.rst:11]: Nitpick:
> investigate _the_ impact

File: docs/scripts/casegen_upcars.rst [hunk 2]
```diff
@@ -2,6 +2,268 @@
...
+fluid flow.
+
+The script can generate various types of grid structure (tilting, dome, and
+hull) amd throws. The input file is in yaml format.
```
Comment (mferrera) [inline docs/scripts/casegen_upcars.rst:15]: Nitpick, maybe?
> grid structure_s_

also 
> s/amd/and

**Review by mferrera — APPROVED** [review]
Highlighted some very minor nitpicks, but great documentation addition 😄👍

**Pull request merged**